data-manipulation/hashing/fnv/hash-data-using-fnv-1.yml

```diff
@@ -0,0 +1,22 @@
+rule:
+  meta:
+    name: hash data using fnv-1
+    namespace: data-manipulation/hashing/fnv
+    author: "@_re_fox"
+    scope: function
+    references:
+      - https://en.wikipedia.org/wiki/Fowler%E2%80%93Noll%E2%80%93Vo_hash_function
+    examples:
+      - ad4229879180e267f431ac6666b6a0a2:0x14007B4D4
+  features:
+    - or:
+      - and:
```

Comment:
potentially could pull this AND up to the top level

Reply:
Good idea, I'll also add in the 32bit version constants

Reply:
Added up the 32b constants, and changed the top level to an `and`